### PR TITLE
reference: fix 6 broken links

### PR DESCRIPTION
**Pull request by CaitinChen**
### What is changed, added or deleted? 

Fix 6 broken links

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs-cn/pull/1846

### Which version does your change affect? 

dev, v2.1, v3.0

### CODE REVIEW

**Comment by CaitinChen** [comment]
@YiniXu9506 FYI.

File: dev/reference/tidb-binlog-overview.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Drainer collects and merges binlogs from each Pump, converts the binlog to SQL o
 
 ### `binlogctl` guide
 
-[`binlogctl`](https://github.com/pingcap/tidb-tools/tree/master/tidb-binlog/binlogctl) is an operations tool for TiDB Binlog with the following features:
+`binlogctl` is an operations tool for TiDB Binlog with the following features:
```
Comment (yikeke) [inline dev/reference/tidb-binlog-overview.md]:
```suggestion
`binlogctl` is an operations tool for TiDB Binlog with the following features:
```
Link `binlogctl` to https://pingcap.com/docs-cn/v3.0/how-to/get-started/tidb-binlog/#binlogctl?

Comment (CaitinChen) [reply dev/reference/tidb-binlog-overview.md]: OK

File: v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md [hunk 2]
```diff
@@ -83,7 +83,7 @@ cd tidb-binlog-kafka-linux-amd64
 
     To guarantee the integrity of data, perform the following operations 10 minutes after Pump is started:
 
-    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/master/tidb-binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
+    - Use binlogctl to generate the `position` for the initial start of Drainer.
```
Comment (yikeke) [inline v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md]:
```suggestion
    - Use [`binlogctl`](/v3.0/how-to/get-started/tidb-binlog.md#binlogctl) to generate the `position` for the initial start of Drainer.
```

File: v2.1/reference/tools/tidb-binlog/tidb-binlog-kafka.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ cd tidb-binlog-kafka-linux-amd64
 
     To guarantee the integrity of data, perform the following operations 10 minutes after Pump is started:
 
-    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/master/tidb-binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
+    - Use [`binlogctl`](/v2.1/how-to/get-started/tidb-binlog.md#binlogctl)) to generate the `position` for the initial start of Drainer.
```
Comment (WangXiangUSTC) [inline v2.1/reference/tools/tidb-binlog/tidb-binlog-kafka.md]:
```suggestion
    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/release-2.1/tidb-binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
```

File: v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md [hunk 4]
```diff
@@ -83,7 +83,7 @@ cd tidb-binlog-kafka-linux-amd64
 
     To guarantee the integrity of data, perform the following operations 10 minutes after Pump is started:
 
-    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/master/tidb-binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
+    - Use [`binlogctl`](/v3.0/how-to/get-started/tidb-binlog.md#binlogctl) to generate the `position` for the initial start of Drainer.
```
Comment (WangXiangUSTC) [inline v3.0/reference/tools/tidb-binlog/tidb-binlog-kafka.md]:
```suggestion
    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/release-2.1/tidb-binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
```

File: dev/reference/tools/tidb-binlog/tidb-binlog-kafka.md [hunk 5]
```diff
@@ -82,7 +82,7 @@ cd tidb-binlog-kafka-linux-amd64
 
     To guarantee the integrity of data, perform the following operations 10 minutes after Pump is started:
 
-    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/master/tidb-binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
+    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/release-2.1/tidb-binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
```
Comment (WangXiangUSTC) [inline dev/reference/tools/tidb-binlog/tidb-binlog-kafka.md]:
```suggestion
    - Use [binlogctl](https://github.com/pingcap/tidb-tools/tree/binlog-kafka/tidb_binlog/binlogctl) of the [tidb-tools](https://github.com/pingcap/tidb-tools) project to generate the `position` for the initial start of Drainer.
```

**Comment by WangXiangUSTC** [comment]
LGTM

**Comment by CaitinChen** [comment]
@yikeke PTAL

**Pull request merged**